pkg/synth/acl.go

```diff
@@ -18,22 +18,20 @@ import (
 const ACLTypeNotSupported = "ACL: src/dst of type %s is not supported."
 
 type ACLSynthesizer struct {
-	spec      *ir.Spec
-	singleACL bool
-	result    *ir.ACLCollection
+	Spec      *ir.Spec
+	SingleACL bool
+	Result    *ir.ACLCollection
 }
 
-// MakeACL translates Spec to a collection of ACLs
+// MakeACL translatea.Spec to a collection of ACLs
```

Comment:
```suggestion
// MakeACL translates Spec to a collection of ACLs
```
OK to push this fix to the following PR